x-pack/legacy/plugins/actions/server/mappings.json

```diff
@@ -29,5 +29,13 @@
         "type": "binary"
       }
     }
+  },
+  "actions-telemetry": {
```

Comment:
we seem to be using underscores instead of hyphens for these property names (eg, `action_type_parameters` above)

Reply:
Don't need this at all, so removed.